crawl-ref/settings/init.txt

```diff
@@ -85,3 +85,6 @@
 # include                 = 0.12_monster_glyphs.txt
 # include                 = 0.13_monster_glyphs.txt
 # include                 = 0.14_monster_glyphs.txt
+
+
+tile_full_screen = false
```

Comment:
To specify local options instead of editing this init.txt (which is the default one and shouldn't have changes committed), either edit the usual one for a local install or run crawl with:

`./crawl -extra-opt-last "tile_full_screen = false"`

when you start it from the command line.

Reply:
what... I thought I manually unchecked those files. thx!